skyportal/models/instrument.py

```diff
@@ -180,6 +180,18 @@ class InstrumentField(Base):
         nullable=False,
     )
 
+    ra = sa.Column(
+        sa.Float,
+        doc='The mid-point right ascension for the tile.',
```

Comment:
Docstring should mention the units. Same for dec.

Reply:
yes.